imports/api/@users/user.js

```diff
@@ -0,0 +1,78 @@
+import { Meteor } from 'meteor/meteor';
+import { Accounts } from 'meteor/accounts-base';
+import SimpleSchema from 'simpl-schema';
+import { Schemas } from '../@/schemas';
+
+Schemas.AccontBase = {
```

Comment:
AccountBase?

Reply:
there is a prebuild Accounts.validateNewUser function from account-base package that validates the new user and then that schema and rest of the user collection merges and validates user collection via Meteor.users ...

Reply:
I meant the typo only :) @ilkergonenc